src/hardware-mapping.cc

```diff
@@ -97,6 +97,21 @@ bool HardwareMapping::AddMotorMapping(LogicAxis axis, int motor,
   return true;
 }
 
+int  HardwareMapping::GetMotorFromAxis(int axis) {
+    int i = 0;
+    // Map back the steps to the correct logical axis.
+    uint8_t temp;
```

Comment:
You can do the assignment in one step to save vertical space
  uing8_t temp = axis_to_driver_[axis];